www/common/common-ui-elements.js

```diff
@@ -970,22 +948,16 @@ define([
                 });
                 break;
             case 'save': // OnlyOffice save
-                button = $('<button>', {
-                    'class': 'fa fa-save',
-                    title: Messages.settings_save,
-                }).append($('<span>', {'class': 'cp-toolbar-drawer-element'})
-                .text(Messages.settings_save))
+                button = createMenuItemButton('fa fa-save', '', Messages.settings_save, Messages.settings_save);
```

Comment:
This is only visible in dev mode so not a big issue, a fix would require a change to the style to avoid applying a margin-right on buttons without text.